contrib/win32/install/install/product.wxs

```diff
@@ -0,0 +1,170 @@
+<?endif?>
+
+<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi"
+  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension"
+  xmlns:fw="http://schemas.microsoft.com/wix/FirewallExtension">
+	<Product
+    Id="$(var.PRODUCTID)"
+    Name="OpenSSH ($(var.CONFIGURATION) $(var.VERSION))"
+    Language="1033"
+    Version="$(var.VERSION)"
+    Manufacturer="$(var.MANUFACTURER)"
+    UpgradeCode="$(var.UPGRADECODE)">
+
+		<Package
+      InstallerVersion="200"
+      Compressed="yes"
+      InstallScope="perMachine"
+      InstallPrivileges="elevated"
+      />
+
+    <Property Id="LOGVERBOSE" Value="LOGVERBOSE"/>
+
+    <!-- https://stackoverflow.com/a/19664484 -->
+    <Upgrade Id="$(var.UPGRADECODEBLOCKED)">
+      <UpgradeVersion OnlyDetect="yes" Property="INCOMPATIBLE_OPENSSH_INSTALLED" Minimum="0.0.0.0" />
+    </Upgrade>
+    <Condition Message="ERROR: OpenSSH debug and release versions cannot be installed side-by-side on the same computer. Please uninstall the existing installation first.">
+      <![CDATA[Installed OR (INCOMPATIBLE_OPENSSH_INSTALLED = "")]]>
+    </Condition>
+
+		<MajorUpgrade AllowSameVersionUpgrades="yes" DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+
+		<MediaTemplate EmbedCab="yes" />
+
+    <UI>
+      <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
+
+      <UIRef Id="WixUI_FeatureTree" />
+      <Publish Dialog="WelcomeDlg" Control="Next" Event="NewDialog" Value="CustomizeDlg">1</Publish>
+      <Publish Dialog="CustomizeDlg" Control="Back" Event="NewDialog" Value="WelcomeDlg">1</Publish>
+    </UI>
+
+    <CustomAction Id="CA_SSHAGENT_PRIVS" Directory="INSTALLFOLDER" Execute="deferred" Impersonate="yes" Return="check" ExeCommand="sc.exe privs ssh-agent SeImpersonatePrivilege" />
+    <CustomAction Id="CA_SSHD_PRIVS" Directory="INSTALLFOLDER" Execute="deferred" Impersonate="yes" Return="check" ExeCommand="sc.exe privs sshd SeAssignPrimaryTokenPrivilege/SeTcbPrivilege/SeBackupPrivilege/SeRestorePrivilege/SeImpersonatePrivilege" />
+
+    <InstallExecuteSequence>
+      <!--https://www.itninja.com/blog/view/appdeploy-articles-msi-advanced-custom-actions?from=appdeploy.com-->
+      <Custom Action="CA_SSHAGENT_PRIVS" After="InstallServices"><![CDATA[&F_Server=3]]></Custom>
+      <Custom Action="CA_SSHD_PRIVS" After="InstallServices"><![CDATA[&F_Server=3]]></Custom>
+    </InstallExecuteSequence>
+    <!--
+    How to enable and disable features refer to
+    https://support.firegiant.com/hc/en-us/articles/230912227-Control-feature-states-for-silent-install-
+    -->
+    <Feature Id="ProductFeature" Title="OpenSSH" Level="1" Display="expand" AllowAdvertise="no">
+			<ComponentGroupRef Id="CG_Main" />
+      <ComponentGroupRef Id="CG_Main_Symbols"/>
+      <Feature Id="F_Client" Title="Client programs" Level="1" Absent="allow" AllowAdvertise="no">
+        <Condition Level="10000">DISABLE_CLIENT</Condition>
```

Comment:
Why using conditions in features instead of just features. If you just want customers to be able to opt into what features they want installed, it's more intuitive - especiually in UI - to just use features.

On the command line, for example, features can be added (if not enable by default) or removed (if enabled by default) like so:

```powershell
msiexec /i openssh.msi 'ADDLOCAL=A;B' REMOVE=C
```

In the UI, you'll actually see a feature tree. The conditions here are extraneous. You could just mark the features as off by default by authoring their level less than the `INSTALLLEVEL` property which is, by default, 1. So if you just mark these as `@Level=2`, they will be off by default.

Reply:
To my shame, I must confess that I did not know this feature of MSIEXEC until now. So we definitly can remove those conditions and simply go with the bare features. But I've to say that I never had issues using the procedure with explicit conditions even in the UI.